Yoast/yoast-components
# Created SettingsSnippetEditor component

## Summary

This PR can be summarized in the following changelog entry:

* Adds `SettingsSnippetEditor` component.

## Test instructions

This PR can be tested by following these steps:

* Link to `wordpress-seo` using `stories/9896-implement-snippet-editor-in-settings-pages` branch in `wordpress-seo` and vistt the `Search appearance` pages of the plugin.
* See if all previews are saved when changed.

Fixes Yoast/wordpress-seo#9896
